assignment1.txt

```diff
@@ -0,0 +1,8 @@
+Albert Zhang
+1. A compiler is what turns the code that is readable by humans into something that the computer can easily read and turn into binary.
+2. A compile error is an error that happens when the computer is compiling the code, and a runtime error is when there is an error when the computer is running the code.
```

Comment:
right, to be more specific, compile errors are usually syntax related, whereas runtime usually has to do with the logic of the code.